packages/mobile/src/backup/BackupPhrase.tsx

```diff
@@ -87,10 +86,26 @@ class BackupPhrase extends React.Component<Props, State> {
     this.setState({
       isConfirmChecked: value,
     })
+
+    CeloAnalytics.track(
+      value
+        ? CustomEventNames.backup_setup_toggle_enable
+        : CustomEventNames.backup_setup_toggle_disable
+    )
   }
 
   onPressConfirmArea = () => {
```

Comment:
do we need the same tracking on both the `onPressConfirmArea` and `onPressConfirmSwitch`? seems kind of redundant but i dont know enough about how the component works to say for sure

Reply:
yes because either might be pressed. 

*Area for when the text is pressed, *Switch for the switch itself. I would like to use one function but Switch component requires a function that takes boolean param

I could de duplicate the logic though

Reply:
ok much better now